app/views/schools/cohorts/show.html.erb

```diff
@@ -0,0 +1,64 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-3"><%= @cohort.start_year %></h1>
+
+    <p class="govuk-body-l govuk-!-margin-bottom-8">
+    <% if @school_cohort.school_chose_cip? %>
+      Training using accredited materials.
+    <% else %>
+      Using a training provider funded by the DfE.
+    <% end %>
+    </p>
+
+    <ol class="app-task-list">
+      <li>
+        <h2 class="app-task-list__section">
+          Next steps
+        </h2>
+        <ul class="app-task-list__items">
+
+          <% if @school_cohort.school_chose_cip? %>
+            <li class="app-task-list__item">
+              <span class="app-task-list__task-name">
+                <%= govuk_link_to "Choose your training materials", "#" %>
+              </span>
+
+              <%= render AutoTagComponent.new(text: @school_cohort.choose_training_materials_status) %>
+            </li>
+          <% else %>
+            <li class="app-task-list__item">
+              <span class="app-task-list__task-name">
+                <%= govuk_link_to "Add estimated numbers of teachers and mentors", "#" %>
+              </span>
+
+              <%= render AutoTagComponent.new(text: @school_cohort.number_of_participants_status) %>
+            </li>
+            <li class="app-task-list__item">
+              <span class="app-task-list__task-name">
+                <%= govuk_link_to "Sign up with a training provider", "#" %>
+              </span>
+
+              <%= render AutoTagComponent.new(text: @school_cohort.training_provider_status) %>
+            </li>
+            <li class="app-task-list__item">
+              <span class="app-task-list__task-name">
+                <%= govuk_link_to "Read and accept privacy and data policy", "#" %>
+              </span>
+
+              <%= render AutoTagComponent.new(text: @school_cohort.accept_legal_status) %>
```

Comment:
Is that a task that will only happen for FIPs? Uhhh, sounds unlikely. 

Not really needed to fix now, just something to flag to Ana / Al